cumulusci/core/config/project_config.py

```diff
@@ -550,40 +590,17 @@ def process_github_dependency(  # noqa: C901
             skip = [skip]
 
         # Initialize github3.py API against repo
-        repo_owner, repo_name = dependency["github"].split("/")[3:5]
-        if repo_name.endswith(".git"):
-            repo_name = repo_name[:-4]
-        gh = self.get_github_api(repo_owner, repo_name)
-        repo = gh.repository(repo_owner, repo_name)
+        repo = self.get_repo_from_url(dependency["github"])
         if repo is None:
             raise DependencyResolutionError(
                 f"{indent}Github repository {dependency['github']} not found or not authorized."
             )
 
+        repo_owner = repo.owner
```

Comment:
@davidmreed can you double check that repo.owner is a string? I'm having a little trouble telling from the github3 docs whether it's a string or an object that returns the username when stringified.

Reply:
It is not a string; it's a `github3.users.ShortUser`. Fixed to explicitly convert.